lighthouse-core/lib/network-recorder.js

```diff
@@ -82,8 +82,14 @@ class NetworkRecorder {
   }
 
   onResourceChangedPriority(data) {
-    this.networkManager._dispatcher.resourceChangedPriority(data.requestId,
-      data.newPriority, data.timestamp);
+    // TODO: this.networkManager._dispatcher.resourceChangedPriority is set to
+    // undefined when onResourceChangedPriority is triggered in
+    // gatherers/offline.js. The underlying cause may need to be investigated.
+    // In the meantime, explicitly check that it's a function.
+    if (typeof this.networkManager._dispatcher.resourceChangedPriority === 'function') {
```

Comment:
I'm not really sure what's up with this, but I could trigger the issue consistently when I was testing against my local 54.0.2825.0 Chrome Canary. Adding in this conditional wrapper didn't seem to break anything else, but I'm at a loss for what this code is supposed to be doing and why it started breaking for me.